app/controllers/api/books_controller.rb

```diff
@@ -15,12 +15,16 @@ def discover
     id_array = current_user.user_books.pluck(:book_id)
     books = Book.where.not(id: id_array)
 
-    render json: books
+    render json: serialize(books)
   end
 
   private
 
   def get_book
     @book = Book.find(params[:id])
   end
+
+  def serialize(data)
+    return BookSerializer.new(data).serializable_hash[:data].pluck(:attributes)
+  end
 end
```

Comment:
can you ensure there's a `new line at end of file`? You can do this by setting it up in vscode

Reply:
like this one?
![image](https://user-images.githubusercontent.com/91857581/190110008-17cb8c84-87ff-4611-a991-55c96a17e335.png)